main.cpp

```diff
@@ -19,13 +19,26 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
-        // Your code goes here
-    };
-
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter) {};
     void run(const MatchFinder::MatchResult &Result) override {
-        // Your code goes here
+        const auto *cast_expr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `cast_expr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
I think I can just add this:
```c++
if (cast_expr == nullptr) {
    return;
}
```